### ISSUE TITLE
New apt release?

### ISSUE THREAD

**Issue by MoffKalast** [UbiquityRobotics/oled_display_node#26]
Do we need to make a new release, so that we can ship the recent changes? Current Noetic beta images still seem to have the outdated version.

**Comment by JanezCim** [comment]
Yeah, I can do a bloom release on our testing mirror if you want. But maybe better thing to do is for someone else to follow the tutorial on how to do it here: https://github.com/UbiquityRobotics/ubiquity_internal/blob/main/apt-bloom-release-tutorial.md and give me feedback for it.
